sound/mpd/Makefile

```diff
@@ -27,16 +27,14 @@ PKG_INSTALL:=1
 PKG_USE_MIPS16:=0
 
 include $(INCLUDE_DIR)/package.mk
-include $(INCLUDE_DIR)/nls.mk
 
 define Package/mpd/Default
   SECTION:=sound
   CATEGORY:=Sound
   TITLE:=Music Player Daemon
   URL:=http://www.musicpd.org/
-  DEPENDS:= +glib2 +libcurl +libpthread +libmpdclient +libstdcpp $(ICONV_DEPENDS) \
-	    +libflac +BUILD_PATENTED:libmad +libvorbisidec +AUDIO_SUPPORT:alsa-lib \
```

Comment:
My general guess (and take on it) would be that -mini emphasizes on size, ffmpeg is about 400kbyte or so larger only handling those formats compared to pulling specific libs. Adding more formats makes ffmpeg a more better choice due to overall size and have better optimization.